kitchen-tests/.kitchen.yml

```diff
@@ -7,6 +7,8 @@ driver:
 
 provisioner:
   name: chef_github
+  chef_omnibus_url: "https://omnitruck.chef.io/current/install.sh"
+  chef_omnibus_install_options: "-n"
```

Comment:
```
$ diff www.install.sh omnitruck.install.sh
565c565
< metadata_url="http://www.chef.io/$channel/$project/metadata?v=$version&p=$platform&pv=$platform_version&m=$machine"
---
> metadata_url="https://omnitruck.chef.io/$channel/$project/metadata?v=$version&p=$platform&pv=$platform_version&m=$machine"
```

Huh. That's kind of confusing. `-n` is backcompat that resolves to `-c current`.

What URL is canonical / most permanent? @sersut?

I presume `chef_omnibus_url` defaults to something like `https://www.chef.io/chef/install.sh`, in which case it would seem like settings `chef_omnibus_install_options` to `-c current` would be the least likely to get deprecated/changed in the future.